### PR TITLE
Project-weather-app, Week 3, Elin Segelöv & Petra Söderström

### PR DESCRIPTION
**Pull request by Petrasoderstrom1612**
https://hilarious-tarsier-00acac.netlify.app/

### CODE REVIEW

**Comment by Petrasoderstrom1612** [comment]
https://hilarious-tarsier-00acac.netlify.app/

**Review by Scarylie — APPROVED** [review]
I think your code and styling looks really great. You should be really proud of yourselves for also acheiving some of the stretch goals.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+✓ The presentation of the data should be in the specified format.
 
-## The problem
+✓ The page should work on mobile (mobile first!), tablet and desktop (Be responsive)
+
+✓ Steps 1 & 2 should be done together and step 3 should be done in branches
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+## The problem
+We fetched data via API from openweather and converted it to JSON. Afterwards we injected data in placeholders using inner HTML. Furthermore we created a filter so that the current weather corresponds to the background as well as the main image and description. Then we worked on creating a weekly weather forecast and polished the project with a responsive design and added additional CSS.
```
Comment (juliaostedt) [inline README.md:14]: Good description of your workflow

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <section class="container">
+            <div class="text-container">
+                <div class="todays-container">
+                    <h1 class="todays what-wheather" id='weather-placeholder'>What's the weather?</h1>
+                    <h1 class="todays" id='temp-placeholder'></h1>
+                    <p class="sunrise-sunset" id="sunrise-time"></p>
+                    <p class="sunrise-sunset" id="sunset-time"></p>
+                    <h2 class="city-name" id="city-placeholder"></h2>
+                    <select name="city-selection" class="city-selection">
+                        <option value="">Select city</option>
```
[the remark this answers is not in the document]
Comment (juliaostedt) [reply index.html:23]: I agree, super nice with an accordion!

File: script.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      })}`;
+      sunsetTime.innerHTML = `sunset ${sunsetStart.toLocaleTimeString([], {
+        hour: "2-digit",
+        minute: "2-digit",
+      })}`;
+
+      // Depending on the weather, icon and background image are displayed
+      //Actual weather & background picture
+      if (json.list[0].weather[0].main.includes("Clouds")) {
+        hero.style.backgroundImage = `linear-gradient(rgba(255,255,255,0.5), rgba(255,255,255,0.5)), url('images/bg-cloud-sun.jpg')`;
```
Comment (juliaostedt) [inline script.js:46]: Nice touch with changing the background image

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+html {
+    box-sizing: border-box;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+#hero {
+  background-image: url(./images/start-hero-img.jpg);  
+  background-size: cover; 
+  background-attachment: fixed;
+  background-repeat: no-repeat;
```
Comment (juliaostedt) [inline styles.css:15]: I have a vague memory of them mentioning that we should refrain from styling IDs but rather add a class to it as well in the HTML. But either way, nice that you found the `background-repeat: no-repeat;` !

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+    padding-bottom: 100px;
+}
+
+.text-container {
+  display: grid;
+  grid-template-columns: 2rem 1fr 1fr 2rem;
+  grid-template-rows: 2rem 1fr 1fr 1fr 2rem;
+  padding: 1rem;
+  height: 80vh;
+  flex-direction: column;
```
Comment (juliaostedt) [inline styles.css:70]: Great job using Grid! Would love a walkthrough if someone's up for it :)

**Review by juliaostedt — COMMENTED** [review]
Great job this week! Impressed with the usage of Grid and the accordion with different cities. As you mentioned, adding the sunrise/sunset times in local time would be cool!

**Pull request closed without merging**